__tests__/api.test.js

```diff
@@ -209,4 +209,53 @@ describe('7. POST /api/reviews/:review_id/comments ', () => {
             expect(res.body.msg).toBe('Bad request')
         })
     });
+});
+
+describe('8. PATCH /api/reviews/:review_id', () => {
+    test('Updates the votes by 1', () => {
+        return request(app)
+        .patch('/api/reviews/1')
+        .send({
+            inc_votes: 1
+        })
+        .expect(201)
+        .then((res) => {
+            expect(res.body.votes).toEqual(2);
```

Comment:
Probably still worth asserting that the right review has been altered - can just check for the review id as well as votes